utils/windows-utils.ts

```diff
@@ -11,4 +14,77 @@ export function validatePackageID(id: string) {
   else {
     throw (`Package ID should be atleast 3 characters long: ${id}`)
   }
+}
+
+const DISALLOWED_WINDOWS_PACKAGE_CHARS_REGEX = /[^a-zA-Z0-9_]/g;
+
+export function generateWindowsPackageId(host: string): string {
+  const parts = host
+    .split(".")
+    .reverse()
+    .map(p => p.trim().toLowerCase())
+    .filter(p => p.length > 0)
+    .map(p => p.replace(DISALLOWED_WINDOWS_PACKAGE_CHARS_REGEX, "-"))
+  parts.push("edge");
+  return parts.join("-");
```

Comment:
Join with period, I'd say. That's how it works for Android, and what most package IDs are for Windows, best I understand.